### PR TITLE
[IMP] l10n: CO content updates

### PR DESCRIPTION
**Pull request by samueljlieber**
This PR improves the content of the CO L10n documentation following the restructure in https://github.com/odoo/documentation/pull/3167.

This PR must follow the merge of #3167 and can be forward ported to 15.0 and 16.0

### CODE REVIEW

**Comment by samueljlieber** [comment]
Updated this PR in https://github.com/odoo/documentation/commit/0ddaa2b389ac861cf82a5b330c2d56501c07ff7f to resemble the restructure in #3167. This PR is ready for your review @StraubCreative 🙂

**Comment by samueljlieber** [comment]
Updated this PR in https://github.com/odoo/documentation/commit/7ae241971a0aefb980dc2080589e7998558577c6 as a second commit to follow the structure JCS implemented in https://github.com/odoo/documentation/pull/3167. I will squash this commit after JCS approves the structure on the related PR.

**Comment by samueljlieber** [comment]
Rebased the merge of the l10n_co restructure and squashed commits and reviewed in https://github.com/odoo/documentation/commit/c4f12218870f90f391fa746516df8d994bcb0459.

File: content/applications/finance/fiscal_localizations/colombia.rst
Comment (toaa-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:14]: Having both a list of :ref:'s *and* the heading menu is redundant

File: content/applications/finance/fiscal_localizations/colombia.rst
Comment (toaa-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:86]:
```suggestion
   - :guilabel:`Fiscal Position`: Determines the taxes/accounts used for this contract.
   - :guilabel:`Tipo de Representación`.
   - :guilabel:`Tipo Establecimiento`.
   - :guilabel:`Obligaciones y Responsabilidades`.
   - :guilabel:`Usuario Aduanero`.
   - :guilabel:`Gran Contribuyente`.
   - :guilabel:`Fiscal Regimen`.
   - :guilabel:`Commercial Name`.
```
Screenshot was in 16/17 but the PR targets 14, there were extra fields in 14 so I added them. I do not know if they are necessary or not, I'll leave that up to you :)

P.S. I also changed the screenshot accordingly

File: content/applications/finance/fiscal_localizations/colombia.rst
Comment (toaa-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:104]:
```suggestion
- :guilabel:`Colombia Template Code`: Select one of the two available templates (:guilabel:`CGEN03`
  or :guilabel:`CGNE04`) to be used in the PDF format of the electronic invoice.
```
I do not know if the template is specified by the Carvajal or not, but if the user is free to choose, it could be nice to add the difference (i.e. the reasons to choose one or the other)

File: content/applications/finance/fiscal_localizations/colombia.rst
Comment (toaa-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:305]:
```suggestion
is correct. Then, proceed to generate a legal XML, which includes a digital signature and a unique
code (CUFE), a PDF invoice that consists of a QR code, and the previously-generated CUFE code.
```
The CUFE section is unclear, is a *2nd* CUFE code generated with the PDF? Or does it use the previously-generated CUFE code and includes that same code with the PDF? I changed the line, but please correct it if necessary

File: content/applications/finance/fiscal_localizations/colombia.rst
Comment (toaa-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:328]:
```suggestion
  - :guilabel:`Full refund and new draft invoice`: Use this option if the credit note is for the
    total invoice and you need the credit note and auto-create a new draft invoice.
```
That sentence is confusing, I'm not sure I understand what it means

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
Hi! Thank you for your hard work, it's a rather long PR. I made some suggestions and comments, some of which could use a clear-up.

General comments are:

- Removed a few .PNGs (that IMO were not necessary)
-  Changed some of the phrasing
- Expanded on some menu pathing
- We no longer :center: pictures (Guidelines may not be updated yet)
- Redundancy between :ref:'s list and heading menu

Please have a look at some of the comments as I made some changes (from what I could understand, but meaning may have gotten lost in translation) :)

File: content/applications/finance/fiscal_localizations/colombia.rst
Comment (samueljlieber) [reply content/applications/finance/fiscal_localizations/colombia.rst:105]: > EWA comment:  The template is defined by Carvajal, and the client determines which one to use with Carvajal. We just sent the data to Carvajal.

File: content/applications/finance/fiscal_localizations/colombia.rst
Comment (samueljlieber) [reply content/applications/finance/fiscal_localizations/colombia.rst]: Updated to include the Odoo description of the field

**Comment by samueljlieber** [comment]
Hi @toaa-odoo thank you for your review and suggestions for improvement. I implemented your improvements and made changes for each of your comments in https://github.com/odoo/documentation/commit/aa99382d484ce7153c2daedde2b92e5e997ce656 🙂

File: content/applications/finance/fiscal_localizations/colombia.rst
Comment (xpl-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:89]: The screenshot is not useful in its current state as it duplicates the info provided above.
Consider removing it, or if you think it's helpful, fill out the fields with a believable example that could help users.

NB: As a general rule, screenshots should give additional information / illustrate something extra or unexpected (e.g., a configuration outside of Odoo, an example of how something looks like after configuration...). In this case, I don't think any screenshot is needed.

Let me reiterate: screenshots are harder to maintain than text; they are not searchable; they are not very visible on portable devices... and most people don't take "good-looking" screenshots, with issues such as being blurry, poorly cropped, taken using different settings (meaning they don't look the same).

Could you revise the whole PR accordingly, as this comment definitely applies to other screenshots here?

**Comment by samueljlieber** [comment]
Hi @xpl-odoo, thank you for your initial review! I've worked with the team to revise this PR. I removed a large number of images in place of descriptive text as well as added additional contextual information where needed in https://github.com/odoo/documentation/commit/3b557e9a8e305f1bd3abddd45a1800406e18cf7f.

**Comment by StraubCreative** [comment]
Rebased on [b676d50](https://github.com/odoo/documentation/pull/4425/commits/b676d50191f53429f3b02553da948459f77e218f)

This is ready for another look, please @xpl-odoo 🙏

**Comment by xpl-odoo** [comment]
> Rebased on [b676d50](https://github.com/odoo/documentation/pull/4425/commits/b676d50191f53429f3b02553da948459f77e218f)
> 
> This is ready for another look, please @xpl-odoo 🙏

Let's proceed as for the MX localization and improve afterward.
@robodoo delegate+

**Comment by samueljlieber** [comment]
@robodoo r+

**Pull request closed without merging**
